erpnext/stock/doctype/pick_list_item/pick_list_item.json

```diff
@@ -187,11 +188,16 @@
    "hidden": 1,
    "label": "Product Bundle Item",
    "read_only": 1
+  },
+  {
+   "fieldname": "barcode",
```

Comment:
This isn't required IMO. 

This was the legacy way to scan barcodes. (Open row > click on barcode field and scan it)

Reply:
Okay, I will remove this. I only added it because I noticed it was in all other doctypes using barcode.